pkg/plugins/common/kustomize/v2-alpha/scaffolds/internal/templates/config/certmanager/kustomizeconfig.go

```diff
@@ -44,20 +44,12 @@ func (f *KustomizeConfig) SetTemplateDefaults() error {
 }
 
 //nolint:lll
-const kustomizeConfigTemplate = `# This configuration is for teaching kustomize how to update name ref and var substitution 
+const kustomizeConfigTemplate = `# This configuration is for teaching kustomize how to update name ref substitution 
 nameReference:
 - kind: Issuer
   group: cert-manager.io
   fieldSpecs:
   - kind: Certificate
     group: cert-manager.io
     path: spec/issuerRef/name
-
-varReference:
-- kind: Certificate
-  group: cert-manager.io
-  path: spec/commonName
-- kind: Certificate
-  group: cert-manager.io
-  path: spec/dnsNames
```

Comment:
@laxmikantbpandhare we are removing the code.
However, are we doing exactly the same behaviour?

Should we not use replacements to ensure the same behaviour without vars?

Reply:
/hold

It is not a bug either. It is an RFE / improvement : sparkles: migrate var for replacements in ....
The goal is we change the scaffold and remove what is deprecated in favor of what should be used instead.